Grammar/python.gram

```diff
@@ -1177,6 +1186,13 @@ invalid_type_param:
                 : "cannot use bound with ParamSpec")
         }
 
+invalid_type_param_expr:
+    | ('*' | '**')? a=expression { 
+            RAISE_SYNTAX_ERROR_KNOWN_LOCATION(
+                a, "expected a type parameter definition, found %s", 
```

Comment:
Should it be called a type parameter expression or type parameter definition here? (I don't remember whether there was a distinction somewhere else).

Reply:
I'd prefer "definition", it's not an expression.